CoreFoundation/PlugIn.subproj/CFBundle_Executable.c

```diff
@@ -119,7 +119,7 @@ static CFURLRef _CFBundleCopyExecutableURLRaw(CFURLRef urlPath, CFStringRef exeN
 #elif DEPLOYMENT_TARGET_WINDOWS
     if (!executableURL) {
         executableURL = CFURLCreateWithFileSystemPathRelativeToBase(kCFAllocatorSystemDefault, exeName, kCFURLWindowsPathStyle, false, urlPath);
-        if (executableURL && !_urlExists(executableURL)) {
+        if (executableURL && !_binaryLoadable(executableURL)) {
```

Comment:
I need to double-check that this matches Darwin behavior. Wednesday.

Reply:
@millenomi - ping on this item!  If this matches the Darwin behavior, I'd like to get this patch merged as well.

Reply:
Spoke offline, this is in fact correct.